### PR TITLE
fix: update point tags [SF-1299]

### PR DESCRIPTION
**Pull request by biga816**
- Change the text of PointTag
- Change `limit` of GetUsersDocument
- Change the description in ReferralCode 
- Remove the Campaign tab 
- Update the referral message
- Update the labels in UserPointInfo

### CODE REVIEW

File: src/utils/env.ts [hunk 1]
```diff
@@ -97,3 +97,10 @@ export const getShowStablecoinAppUrl = () => {
     }
     return NEXT_PUBLIC_SHOW_STABLECOIN_APP_URL === 'true';
 };
+
+export const getReferralMessage = (): string => {
+    const referralMessage = process.env.NEXT_PUBLIC_REFERRAL_MESSAGE;
+
+    assert(referralMessage, 'NEXT_PUBLIC_REFERRAL_MESSAGE is not set');
```
Comment (as1602) [inline src/utils/env.ts:104]: let's not throw an error here, if it's not a roadblocker to the application functionality.

File: src/components/organisms/Header/constants.tsx [hunk 2]
```diff
@@ -27,12 +27,6 @@ export const PRODUCTION_LINKS = [
         dataCy: 'points',
         className: 'flex laptop:hidden desktop:flex',
     },
-    {
```
Comment (as1602) [inline src/components/organisms/Header/constants.tsx:30]: we don't want to show the campaign page anymore?

Comment (biga816) [reply src/components/organisms/Header/constants.tsx:30]: Yes, it's Masa's requirement.
https://securedfinance.slack.com/archives/C032N8KUNRZ/p1735090129530299?thread_ts=1734657573.965039&cid=C032N8KUNRZ

Comment (as1602) [reply src/components/organisms/Header/constants.tsx:30]: can you remove this please from links.tsx?
{
        text: 'Campaign',
        href: '/campaign',
        icon: <GiftIcon className='h-5 w-5' />,
        isInternal: true,
    },

Comment (biga816) [reply src/components/organisms/Header/constants.tsx:30]: I removed it!

File: src/utils/env.ts (hunk 1 above)
Comment (biga816) [reply src/utils/env.ts]: I fixed that!

File: src/utils/links.test.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { LinkList } from './links';
 
 describe('links', () => {
-    it('should have eight links', () => {
-        expect(LinkList).toHaveLength(8);
+    it('should have seven links', () => {
+        expect(LinkList).toHaveLength(7);
```
[the remark this answers is not in the document]
Comment (biga816) [reply src/utils/links.test.tsx:5]: The Campaign page should not be deleted as it may be reused in the future.

**Review by as1602 — APPROVED** [review]

**Pull request merged**